bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -542,4 +532,18 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
 
     {% endfor -%}
     {% endif -%}
+
+    {% if task.depends_on_fivetran != None -%}
+    {% for fivetran_task in task.depends_on_fivetran -%}
```

Comment:
There's already the same check and loop over `task.depends_on_fivetran` above, so I think the contents of this outer loop could be moved inside that existing loop.